packages/x-date-pickers/src/internals/models/validation.ts

```diff
@@ -124,3 +124,14 @@ export interface DateTimeValidationProps<TDate extends PickerValidDate> {
    */
   maxDateTime?: TDate;
 }
+
+/**
+ * Props used for range validation.
+ */
+export interface DateRangeValidationProps<> {
+  /**
+   * If `true`, throws an error when the selected range that contains disabled dates.
+   * @default false
+   */
+  disableNonContiguousRanges?: boolean;
```

Comment:
Isn't having this prop used in fields and Pickers enough?
We should strive to have a single source of truth for prop definitions if possible.